internal/docker/builder.go

```diff
@@ -328,6 +328,10 @@ func (d *Builder) construct(bprint b.Blueprint) (errs []error) {
 		// Log again so we can see the timings.
 		d.log("%s: Stopped container: %s", res.contextStr, res.containerID)
 
+		for _, c := range changes {
+			d.log(c)
+		}
```

Comment:
debug code left in? If you want to keep this, probably want to prefix the log line with something meaningful

Reply:
thanks, fixed :)